src/qt/rpcconsole.cpp

```diff
@@ -1186,6 +1198,11 @@ void RPCConsole::showEvent(QShowEvent *event)
 
 void RPCConsole::hideEvent(QHideEvent *event)
 {
+    // It is too late to call QHeaderView::saveState() in ~RPCConsole(), as all of
```

Comment:
An alternative could be to use `QHeaderView::sectionResized` signal.

Reply:
Yes, using `QHeaderView::sectionResized` signal is more clear solution. But most of them are redundant, as we are interested in final state only.

Do you think we should prefer clear code (I admit that using a comment to explain code is a bad sign)?
If so, I'll be happy to implement your suggestion.

Reply:
On the other hand with current approach it always save even if you don’t resize.

Is the signal emitted while resizing? Not only at the end? (Didn’t test)

Reply:
@promag
> Is the signal emitted while resizing? Not only at the end? (Didn’t test)

Signal is emitted while resizing, at least on Linux Mint 20.1 + Qt 5.12.8.